pkg/virt-controller/services/rendervolumes.go

```diff
@@ -368,7 +368,7 @@ func withBackendStorage(vmi *v1.VirtualMachineInstance) VolumeRendererOption {
 			return nil
 		}
 
-		volumeName := vmi.Name + "-state"
+		volumeName := "backendstorage-state"
```

Comment:
Is this used for vTPM only, or also for NVRAM?

If vtpm only, could we name it `vtpm-state`?

Reply:
The volume is shared with nvram. I am open to new suggestions if name does not perfectly fit :)

Reply:
firmware-state ?

Reply:
IDK if it fits better actually. I would prefer `backendstorage-state` or simply `backendstorage` (it's the same for me). It will help users with mental link to the topic. But I don't have a strong opinion.
So if you strongly think that `backendstorage-state` is wrong I can change it :)

Reply:
I wonder how many users will understand what backendstorage will mean.

but, actually, this is on the pod level only, thus ... I probably does not matter much.

Consider it resolved, take what you need.

Reply:
There's already been a discussion about how bad the name backendstorage is, and we should try to change it at some point (my favorite replacement name was something like "emulation data").
For this PR though, I agree that we should reuse the existing conventions. For the CR setting, I actually use the much better "vm-state", I suggest we use that here too. Or "backendstorage", but "backendstorage state" doesn't make sense to me.

Anyway, thank you for the fix @fossedihelm !
/approve
/hold

Reply:
Let's move to `emulation-data`?

Reply:
Ok, moved to `vm-state` :D